src/commands/base-command.js

```diff
@@ -437,27 +430,10 @@ class BaseCommand extends Command {
     const globalConfig = await getGlobalConfig()
     const { NetlifyAPI } = await jsClient
 
-    const configWatcher = new events.EventEmitter()
-
-    // Only set up a watcher if we know the config path.
-    if (configPath) {
```

Comment:
What do you think about keeping this in the base command, and only applying for `dev`?
I'm thinking we might want this for `functions:serve` and possibly other long running commands

Reply:
I'd prefer to do that work when we need that for `functions:serve`. There are (as evidenced by this PR) wrinkles to figure out wrt memory usage and I believe exposing this in `BaseCommand` could give false confidence that this is ready for wide application.